telemetry/specs/INT.mdk

```diff
@@ -1253,9 +1290,11 @@ INT Metadata Header and Metadata Stack:
     0                   1                   2                   3
     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-   | Ver=1 | 0 |0|0|0|     Reserved      | HopML=2 |RemainingHopC=5|
+   | Ver=2 | 0 |0|0|0|0|   Reserved      | HopML=2 |RemainingHopC=5|
```

Comment:
I don't see any definition of a new flag bit after the "M" bit.
The only change here should be "Ver=1" --> "Ver=2".

Reply:
Removed the new flag bit. We need to discuss the need for a bit to indicate the VxLAN vs VxLAN GPE encap of the incoming packet. Do we need that indication in the header for the sink node processing.

Reply:
We're addressing the GPE discussion and flag in #58. 
Let's merge this PR first into mater and rebase #58.